### PR TITLE
Introduced a new folder structure for each API version

### PR DESCRIPTION
**Pull request by abdallahbeshi**
## Description

Introduced a new folder structure for each API version.

image

## Type of change
- [ ] Bug fix.
- [ ] Feature.
- [x] Consolidation.

## Checklist
- [ ] Is breaking change.
- [ ] Documentation updated.
- [ ] Tests included (please specify cases).

### CODE REVIEW

File: src/Germany/Mews.Fiscalizations.Germany.Tests/TestFixture.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿using Mews.Fiscalizations.Germany.Model;
+﻿using Mews.Fiscalizations.Germany.V1;
+using Mews.Fiscalizations.Germany.V1.Model;
```
Comment (KaliCZ) [inline src/Germany/Mews.Fiscalizations.Germany.Tests/TestFixture.cs:2]: This is weird. Why do we now need more usings? :D

File: src/Germany/Mews.Fiscalizations.Germany.Tests/TestFixture.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 using System;
 
 namespace Mews.Fiscalizations.Germany.Tests
```
Comment (KaliCZ) [inline src/Germany/Mews.Fiscalizations.Germany.Tests/TestFixture.cs:5]: We could also copy paste the tests, so that tests have a v1 too.

Comment (abdallahbeshi) [reply src/Germany/Mews.Fiscalizations.Germany.Tests/TestFixture.cs:5]: I would keep tests for the latest version only.

Having tests for all versions means we have to add more pipelines and maintain all of them.

File: src/Germany/Mews.Fiscalizations.Germany.Tests/TestFixture.cs (hunk 1 above)
Comment (abdallahbeshi) [reply src/Germany/Mews.Fiscalizations.Germany.Tests/TestFixture.cs:2]: `FiskalyClient` is no longer in Mews.Fiscalizations.Germany
it is now Mews.Fiscalizations.Germany.V1 so we must add the using.

the other using was there before.

**Review by KaliCZ — APPROVED** [review]

**Pull request merged**